modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderUploadCSV.testcase

```diff
@@ -60,4 +60,86 @@ definition {
 			ImportExport.assertDownloadedFileName(downloadedFile = "csv_template.csv");
 		}
     }
+
+	@description = "Verify user can not upload non CSV files"

```

Comment:
@AfonsoHSantos , please check the file names uploaded in this test. Example, `Document_1.pdf` etc. In another commit, the files added are `Commerce.pdf`, etc